headers/xdp/libxdp.h

```diff
@@ -15,6 +15,14 @@
 #include <bpf/bpf.h>
 #include "xdp_helpers.h"
 
+#ifndef LIBXDP_API
+#define LIBXDP_API __attribute__((visibility("default")))
```

Comment:
You are right, my mistake.
I (erroneously) though libxdp was following a similar pattern as libbpf.
Btw, don't you think having the default visibility set to hidden would be the right way to go?

Reply:
We do the reverse (see `LIBXDP_HIDE_SYMBOL` in libxdp_internal.h). There are only a few internal functions, so didn't seem worth it to clutter the main include with lots of defines when we could just hide the few we don't want to expose... Why do you think it would be better to default do the reverse?

Reply:
I think it is just a different choice. IMHO, manually allowing the symbols we want gives us more control on what we really export, instead of hiding the ones that we don't want. But again, both ways are absolutely correct. So, I am fine with the current approach.

If it is ok for you, I will change the PR to just leave the `extern C`. Thanks!

Reply:
Sure, please do!

Reply:
Done. Thanks!